components/Table/Table.js

```diff
@@ -95,21 +152,139 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  createFilterMenu() {
+    const filterMenuContainer = document.createElement('div');
+    filterMenuContainer.classList.add('filter-menu-container');
+
+    const filterLabel = document.createElement('div');
+    filterLabel.classList.add('filter-label');
+    filterLabel.textContent = 'Name';
+
+    const filterOperator = document.createElement('div');
+    filterOperator.classList.add('filter-operator');
+    filterOperator.textContent = 'equal';
+
+    const filterValue = document.createElement('input');
+    filterValue.classList.add('filter-value');
+    filterValue.placeholder = 'Filter value';
+
+    const btnClean = document.createElement('div');
+    btnClean.classList.add('btn-clean');
+    btnClean.textContent = 'x';
+
+    const btnExit = document.createElement('div');
+    btnExit.classList.add('btn-exit');
+    btnExit.textContent = 'x';
+
+    btnExit.addEventListener('click', () => {
+      filterMenuContainer.parentElement.children[1].remove();
+    });
+
+    btnClean.addEventListener('click', () => {
+      filterValue.value = '';
+      document
+        .querySelectorAll('.table-row')
+        .forEach((row) => (row.style.display = 'flex'));
```

Comment:
Seems like you forgot to fix this part
Currently, if we click on the `clean` button - nothing happens